release.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -euo pipefail
+
+NEW_VERSION="$(date +%Y.%m.%d)"
+OLD_VERSION="$(git tag --list | tail -n 1)"
```

Comment:
TL;DR All is well, and I learned something.

Interesting aside: this command returns the same value even if you have an old commit checked out. I was wondering if the ordering of this command and the `git fetch -p --all; ...` block below matters, and it seems the ordering doesn't matter.